### PR TITLE
Fixes #78 - codeowners correct structure

### LINKED ISSUE #78: FIx CODEOWNERS
The sub-folders for code ownership are not working as intended.

### PR DESCRIPTION
**Pull request by sudermanjr**

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -1 +1,9 @@
+stable/rbac-manager @sudermanjr @robscott
+stable/aws-iam-authenticator @sudermanjr @davekonopka
+incubator/autospotting @sudermanjr @coreypobrien
+incubator/fluentd @dosullivan @coreypobrien
+incubator/helm-release-pruner @davekonopka
+incubator/imagepullsecret-deploy @crmejia
+incubator/kubebench @mjhuber
+incubator/ro-cert-manager @sudermanjr @endzyme
```
Comment (davekonopka) [inline CODEOWNERS]: The [Github docs](https://help.github.com/en/articles/about-code-owners) seem to say that broad default should go first followed by specific overrides:

```# These owners will be the default owners for everything in
# the repo. Unless a later match takes precedence,
# @global-owner1 and @global-owner2 will be requested for
# review when someone opens a pull request.```

Comment (davekonopka) [inline CODEOWNERS:8]: The [Github docs](https://help.github.com/en/articles/about-code-owners) seem to suggest the opposite order: broad first followed by more specific overrides.

Comment (sudermanjr) [reply CODEOWNERS:8]: I read that the opposite.  Doing this in another repo earlier seems to confirm that, but I will double check

**Comment by coreypobrien** [comment]
Didn't realize this was already open! When I accidentally redid this in #81 I had a longer list of projects too. Any reason to exclude those?

**Comment by sudermanjr** [comment]
I left out projects where Dave or I were the only codeowners since we are already entire project codeowners.  I don't really care which one we merge.

**Pull request closed without merging**